go.mod

```
@@ -1,19 +1,19 @@
 module github.com/openebs/zfs-localpv
 
-go 1.19
+go 1.20
 
 require (
 	github.com/container-storage-interface/spec v1.8.0
 	github.com/kubernetes-csi/csi-lib-utils v0.9.0
-	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.27.7
+	github.com/onsi/ginkgo/v2 v2.19.0
+	github.com/onsi/gomega v1.33.1
```

Good idea to bump gomega as well
```suggestion
	github.com/onsi/gomega v1.34.1
```